sufia-models/lib/generators/sufia/models/templates/config/sufia.rb

```diff
@@ -105,8 +105,11 @@
   # If you have ffmpeg installed and want to transcode audio and video uncomment this line
   # config.enable_ffmpeg = true
 
-  # Specify the Fedora pid prefix:
-  # config.id_namespace = "sufia"
+  # Sufia uses NOIDs for files and collections instead of Fedora UUIDs
```

Comment:
It might be good to clarify that NOIDs are in the form "nnnn" where as Fedora UUIDs are in the form "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx". Not sure how much common knowledge this is.